docker/measure-worker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/fuzzbench/base-runner
+
+ENV WORK /work
+WORKDIR $WORK
+
+ENV VIRTUALENV_DIR $WORK/.venv
+
+RUN mkdir -p /work && virtualenv --python=$(which python3) $VIRTUALENV_DIR
```

Comment:
/work -> $WORK

Reply:
python3 -m venv ?

Reply:
>/work -> $WORK

Done.

>python3 -m venv ?

Spoke to laszlo about this in the original PR. I'll change all uses to venv at once rather than have this one be inconsistent.